api/server/middleware/experimental.go

```diff
@@ -0,0 +1,29 @@
+package middleware
+
+import (
+	"net/http"
+
+	"golang.org/x/net/context"
+)
+
+// ExperimentalMiddleware is a the middleware in charge of adding the
+// 'Docker-Experimental' header to every outgoing request
+type ExperimentalMiddleware struct {
```

Comment:
Doesn't `golint` errors out on this :sweat_smile: ? `middleware.ExperimentalMiddleware` seams a bit redondant :angel: :stuck_out_tongue_closed_eyes: